### PR TITLE
feat: created natayaprice folder for ReadMe in html

### PR DESCRIPTION
**Pull request by NatayaP**
created natayaprice folder for ReadMe in html

### CODE REVIEW

File: lesson_15/natayaprice/ts-node-express/src/index.ts
Comment (anthonydmays) [inline lesson_15/natayaprice/ts-node-express/src/index.ts:1]: Seems that you're missing your readme files and the correct code for hosting them.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Comment by NatayaP** [comment]
I read through the articles and started over a few times, but I'm facing an issue when I run certain commands. It's broken, but all my files are there.

File: lesson_15/natayaprice/ts-node-express/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// src/index.js
+import express, { Express, Request, Response } from "express";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const app: Express = express();
+const port = process.env.PORT || 3000;
+
+app.get("/", (req: Request, res: Response) => {
```
Comment (anthonydmays) [inline lesson_15/natayaprice/ts-node-express/src/index.ts:10]: Read the articles on hosting static files using express again.

File: lesson_15/natayaprice/ts-node-express/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "1.0.0",
+  "main": "dist/index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
```
Comment (anthonydmays) [inline lesson_15/natayaprice/ts-node-express/package.json:12]: Also looks like you are missing dependencies.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_15/natayaprice/ts-node-express/src/index.ts (hunk 1 above)
Comment (NatayaP) [reply lesson_15/natayaprice/ts-node-express/src/index.ts:10]: I changed line 10 to app.use and added the missing dependencies, but when I try to run this command: "npx ts-node src/index.ts" I'm receiving this error: "error TS2307: Cannot find module 'dotenv' or its corresponding type declarations." So, I believe it's still broken and I'm unsure of where/how to fix it.

**Pull request closed without merging**